kubevirt-101/step2.md

```diff
@@ -0,0 +1,80 @@
+#### Deploy a VM
+
+The command below applies a yaml definition of a virtual machine into our current Kubernetes environment, defining the vm name, the resources required (disk, cpu, memory), etc. You can take a look to the [vm.yaml](https://raw.githubusercontent.com/kubevirt/demo/master/manifests/vm.yaml) file if you have interest in knowing more about a virtual machine definition:
+
+`kubectl apply -f https://raw.githubusercontent.com/kubevirt/demo/master/manifests/vm.yaml`{{execute}}
+
+We are creating a Virtual Machine in the same way as we would create any other Kubernetes resource thanks to what KubeVirt has enabled in our environment. Now we have got a Virtual Machine as a Kubernetes resource.
+
+After the vm resource has been created, you can manage the VMs with standard 'kubectl' commands:
+
+```
+$ kubectl get vms
+$ kubectl get vms -o yaml testvm
+```
+
+Check VM's defined (using commands above)
+
+`kubectl get vms`{{execute}}
+
+To start a VM, virtctl should be used:
+
+`./virtctl start testvm`{{execute}}
+
+Now, you can check again the vm status:
+
+`kubectl get vms`{{execute}}
+
+Alternatively you can use `kubectl edit vm testvm` to set `.spec.running: true`.
+
+Once the VM is running you can inspect its status as [instance](https://kubevirt.io/user-guide/docs/latest/creating-virtual-machines/intro.html) :
```

Comment:
I believe this is not clear enough, thinking in someone who sees KubeVirt for the first time. I would expand this a bit more:

> A `VirtualMachine` resource contains a VM's definition and status. An instance of a running VM has an additional associated resource: a `VirtualMachineInstance`.
>
> Once the VM is running you can inspect its instance's status: